source/integration-guide/order-confirmation.rst

```diff
@@ -206,7 +206,7 @@ sizeAlias
         // When the size is not understandable by customers, like:
         size: "2148"
         // Then the sizeAlias can be use to specify what the customer will see for
-        // this product in his purchase history the next time they open the widget:
+        // this product in their purchase history the next time they open the widget:
```

Comment:
Also here. `customer` is singular, so it should be `his`, or am I missing something?!